### DISCUSSION TITLE
In RainbowKit modal, I see all wallet options on localhost but in production I only see a few options

### DISCUSSION THREAD

**Discussion by stuckinaboot** [rainbow-me/rainbowkit#1900] [Q&A] [answered]
I followed the RainbowKit setup guide (https://www.rainbowkit.com/docs/installation) and created a project id in cloud.walletconnect

Anybody have suggestions for how to resolve this issue? Ideally was trying to stick with Rainbowkit for now unless it's rly not possible to fix

```
"use client";

import * as React from "react";
import {
  RainbowKitProvider,
  getDefaultWallets,
  getDefaultConfig,
} from "@rainbow-me/rainbowkit";
import {
  argentWallet,
  trustWallet,
  ledgerWallet,
  walletConnectWallet,
} from "@rainbow-me/rainbowkit/wallets";
import { base, baseSepolia } from "wagmi/chains";
import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
import { WagmiProvider } from "wagmi";
import { testnetsEnabled } from "./constants";

const { wallets } = getDefaultWallets();

const config = getDefaultConfig({
  appName: "WillieNet Dapp",
  // I hardcode this in my project to the correct value from walletconnect cloud but replaced with 'xxx' here for privacy
  projectId: "xxx",
  wallets: [
    ...wallets,
    {
      groupName: "Other",
      wallets: [walletConnectWallet, argentWallet, trustWallet, ledgerWallet],
    },
  ],
  chains: testnetsEnabled ? [baseSepolia] : [base],
  ssr: true,
});

const queryClient = new QueryClient();

export function Providers({ children }: { children: React.ReactNode }) {
  return (
    <WagmiProvider config={config}>
      <QueryClientProvider client={queryClient}>
        <RainbowKitProvider>{children}</RainbowKitProvider>
      </QueryClientProvider>
    </WagmiProvider>
  );
}

```

Screenshot of running on localhost
<img width="739" alt="image" src="https://github.com/rainbow-me/rainbowkit/assets/1824753/3ffe4444-da81-4e1c-b9a1-6d28c5fa3a09">

Screenshot of running in prod
<img width="739" alt="image2" src="https://github.com/rainbow-me/rainbowkit/assets/1824753/ce2c8c26-e3a5-4bf3-9e1d-104ac0b7c525">

**Comment by 0xmkh** [comment]
@stuckinaboot Can you share a simple a small reproducible example with a github repo code or codesandbox ?

**Reply by 0xmkh** [reply]
Also can you double check the production environment if you use the same code or not ? Just check if it's linked to the correct branch where you have all your wallets included.

**Reply by stuckinaboot** [reply]
Sounds good! Yeah I'll try to repro in a smaller new repo and will share here

**Reply by 0xmkh** [reply]
Can you try downgrading your `wagmi` version to `2.5.12` ? Just found out it's a bug related to the latest wagmi version.

**Reply by DanielSinclair** [reply, marked as answer]
@stuckinaboot Can confirm that downgrading to Wagmi `2.5.12` resolved this issue when testing. We'll get that sorted out with the Wagmi team shortly.

**Reply by SimonVillage** [reply]
Damn, that took us a while to debug ...

**Reply by DanielSinclair** [reply]
@SimonVillage Apologies about that! This is now resolved in the latest version of Wagmi
